src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -720,6 +720,13 @@ static constexpr Property<float, PropertyMutability::RO> thermal{"DEVICE_THERMAL
  */
 static constexpr Property<std::vector<std::string>, PropertyMutability::RO> capabilities{"OPTIMIZATION_CAPABILITIES"};
 
+/**
+ * @brief Read-only property to get a std::vector<PropertyName> of properties which should affect
+ * the hash calculation for model cache
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> caching_properties{"CACHING_PROPERTIES"};
```

Comment:
If we really need this property, need to hide from public API

Reply:
The property has been moved to internal plugin config, thanks